src/mempool.rs

```diff
@@ -0,0 +1,71 @@
+use std::ops::Index;
+use hyper::{Body, Client, Uri};
+use serde_derive::Deserialize;
+use serde_derive::Serialize;
+
+type ConnectivityResponse = Vec<Nodes>;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct Nodes {
+    pub public_key: String,
+    pub channels: i64,
+    pub capacity: i64,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct Node {
+    pub sockets: String,
+}
+
+// TODO return result here
+async fn body_to_string(body: Body) -> String {
+    let body_bytes = hyper::body::to_bytes(body).await.unwrap();
+    String::from_utf8(body_bytes.to_vec()).unwrap()
+}
+
+pub async fn get_recommended_channels() -> Result<Vec<String>, hyper::http::Error> {
+    let https = hyper_tls::HttpsConnector::new();
+    let client = Client::builder()
+        .build::<_, hyper::Body>(https);
+    let uri: Uri = "https://mempool.space/api/v1/lightning/nodes/rankings/connectivity".parse().unwrap();
+    let res = client.get(uri).await.unwrap();
+
+    // Recommend 6 high liquidity channels
+    // Recommend 4 smaller edge nodes
```

Comment:
I am thinking that this too many to start with (we ourselves have only tested 2 channels afaik).

How do you feel about suggesting 3:
1x Routing node
1x edge node
1x Nolooking special suggestion node (e.g. HRF)

Reply:
(im writing a commit to fetch nolooking special suggested node)